subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/plotting/perf_report.py

```diff
@@ -29,21 +29,24 @@ def __init__(self):
 
     def do_plot(self, *args):
         ordered_vars, settings, setting_lists, variables, cfg = args
-        if sum(1 for _ in common.Matrix.all_records(settings, setting_lists)) != 1:
+        
+        cnt = common.Matrix.count_records(settings, setting_lists, include_lts=True)
+
+        if cnt != 1:
             return {}, "ERROR: only one experiment must be selected"
 
-        for entry in common.Matrix.all_records(settings, setting_lists):
-            pass
+#        for entry in common.Matrix.all_records(settings, setting_lists):
+#            pass
 
         header = []
 
         header += [html.P("This report shows the RHODS/Notebook performance testing setup and results.")]
         header += [html.H1("Performance Report")]
 
-        setup_info = error_report._get_test_setup(entry)
-        header += [html.Ul(
-            setup_info
-        )]
+#        setup_info = error_report._get_test_setup(entry)
+#        header += [html.Ul(
+#            setup_info
+#        )]
```

Comment:
not sure why you removed this block and the one above,
I included the test setup here to have a "standalone performance report", ie the test setup and the key performance results in a single page, for easy sharing

Reply:
Thanks for catching, I commented this out since there is no PR information with LTS payloads presently.